typing/subst.ml

```diff
@@ -155,12 +155,12 @@ let rec typexp copy_scope s ty =
           else newty2 ty.level desc
         in
         For_copy.save_desc copy_scope ty desc;
-        Private_type_expr.set_desc ty (Tsubst ty');
+        Private_type_expr.set_desc ty (Tsubst (ty', None));
         (* TODO: move this line to btype.ml
            there is a similar problem also in ctype.ml *)
         ty'
       else ty
-  | Tsubst ty ->
+  | Tsubst (ty, _) ->
```

Comment:
Can we assert = None here?

Reply:
This case actually failed at one test (poly.ml).  I am now looking into the bug..

Reply:
This is needed when the row variable of a polymorphic variant is `Tunivar`.